diagnostic.md

```diff
@@ -29,50 +30,59 @@ Why don't we use views in our interpretation of the MVC pattern?
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create, read, update, destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+delete, show, update, destroy, create
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+-Browser parses url
+-browser sends request to the server, usually with http request (get)
+-server sends to the appropriate controller
+-controller gathers data from models and returns to server
+-server returns information back to the user's browser
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+rails new <application-name> --api
 ```
 
 What is the command to start an instance of a rails server?
 
 ```bash
-// your response here
+bin/rails server
 ```
 
 What are the commands to drop, create, migrate and seed a database from the command
 line? (5 bullet points)
 
 ```bash
-// your response here
+rake db:drop
+rake db:create
+rake db:migrate
+db:test:prepare
+rake db:seed
 ```
 
 What is the command to scaffold a pet with a name and age attributes (hint:
 Also think of the data types for each attribute)?
 
 ```bash
-// your response here
+Pet > rails generate scaffold Post name:string age:integer
+
```

Comment:
Very close here Joni, should be:

`rails generate scaffold pet name:string age:integer`